```
48%] Building CXX object examples/MatPowerTesting/CMakeFiles/test_parse_gen_row.dir/test_parse_gen_row.cpp.o
In file included from /Users/szb/GridKit_dir/GridKit/examples/MatPowerTesting/test_parse_gen_row.cpp:5:
/Users/szb/GridKit_dir/GridKit/src/Utilities/Testing.hpp:235:30: error: no template named 'map' in namespace 'std'; did you mean 'max'?
  235 |     inline bool isEqual(std::map<IdxT, RealT> a,
      |                         ~~~~~^~~
      |                              max
/opt/homebrew/Cellar/llvm/19.1.7/bin/../include/c++/v1/__algorithm/max.h:29:1: note: 'max' declared here
   29 | max(_LIBCPP_LIFETIMEBOUND const _Tp& __a, _LIBCPP_LIFETIMEBOUND const _Tp& __b, _Compare __comp) {
      | ^
In file included from /Users/szb/GridKit_dir/GridKit/examples/MatPowerTesting/test_parse_gen_row.cpp:5:
/Users/szb/GridKit_dir/GridKit/src/Utilities/Testing.hpp:235:17: error: redefinition of 'isEqual' as different kind of symbol
  235 |     inline bool isEqual(std::map<IdxT, RealT> a,
      |                 ^
/Users/szb/GridKit_dir/GridKit/src/Utilities/Testing.hpp:207:17: note: previous definition is here
  207 |     inline bool isEqual(PowerSystemData::SystemModelData<RealT, IdxT> a,
      |                 ^
/Users/szb/GridKit_dir/GridKit/src/Utilities/Testing.hpp:235:47: error: expected ')'
  235 |     inline bool isEqual(std::map<IdxT, RealT> a,
      |                                               ^
/Users/szb/GridKit_dir/GridKit/src/Utilities/Testing.hpp:235:24: note: to match this '('
  235 |     inline bool isEqual(std::map<IdxT, RealT> a,
      |                        ^
/Users/szb/GridKit_dir/GridKit/src/Utilities/Testing.hpp:237:91: error: expected ';' after top level declarator
  237 |                         const RealT           tol = std::numeric_limits<RealT>::epsilon())
      |                                                                                           ^
      |                                                                                           ;
4 errors generated.
```

--- a/src/Utilities/Testing.hpp
+++ b/src/Utilities/Testing.hpp
@@ -10,6 +10,8 @@
 #include <cmath>
 #include <iostream>
 #include <limits>
+#include <map>
+#include <vector>
 
 #include <PowerSystemData.hpp>
 

